stackrox-container-image-scanner/pom.xml

```diff
@@ -14,6 +14,9 @@
     <properties>
         <jenkins.version>2.164.1</jenkins.version>
         <java.level>8</java.level>
+        <jquery.version>3.4.1</jquery.version>
+        <datatables.version>1.10.20</datatables.version>
+        <bootstrap.version>4.4.1</bootstrap.version>
```

Comment:
I understand these versions are the ones that were previously hardcoded in the pom.xml file and not kept aligned by the dependabot bumps.
IMHO, it would be good to have a follow-up PR to bring these versions to date with the latest dependabot state (and possibly to keep watch for version bumps, which seem to be pretty infrequent).

Reply:
https://issues.redhat.com/browse/ROX-28781